### ISSUE TITLE
Synthesised output audio and provide text doesn't match

### ISSUE THREAD

**Issue by shwetagargade216** [r9y9/deepvoice3_pytorch#139]
I did 16000 steps for speaker adaption on own data, But synthesised audio and provided sentences in  text_list doesn't match. 
I did everything on master, used previous ljspeech checkpoints, and trained model for speaker adaption.

**Comment by G-Wang** [comment]
What do you mean by doesn't match? Does it produce words that you can understand? Or it's not trained properly and it's just muttering? Do you have samples that you can share?

What is the dataset? How much data are you using? Did you trim silence before and after each audio sample?

We'll need more info to understand what's wrong.

**Comment by shwetagargade216** [comment]
I mean, after synthesising I'm getting audio file in provided path, but audio lyrics dosen't match with sentences which were I provided in text_list.txt
I ran this..
python synthesis.py --preset=presets/nyanko_ljspeech.json  /malenyanko/checkpoint_step000016200.pth  /text_list.txt  /nyanko1 
Audio quality is really good, But I want provide sentences For example:
Generative adversarial network or variational auto-encoder.
depp speech is really awesome

In audio I'm getting different sentences. Random sentences.

**Comment by shwetagargade216** [comment]
Yes I', able listen whatever in audio. Sound quality is also good. 
I took cmu_arctic male data , 150 samples, and I did 16900 iterations with default parameters. 

Samples[https://drive.google.com/drive/u/0/folders/1XIoV-Apfda35JuPfTckuIVS_KWtSKbLp](url)

**Comment by shwetagargade216** [comment]
Why synthesised audios taking random sentences ?

**Comment by shwetagargade216** [comment]
@G-Wang @r9y9 any comments ?

**Comment by r9y9** [comment]
I remember that I was having difficulty to learn char embedding correctly for the nyanko model (i,e, as a failure case, a model can ignore input text). I suspect that might be due to the deeper architecture of the nyanko model. Not really sure about the reason why you have a (possibly similar) problem. I'm still not exactly sure what problems you see though. As @G-Wang says, we will need more information to help you out.

**Comment by shwetagargade216** [comment]
@r9y9  Thanks for the reply. 
I did't changed any parameters, Followed Master readmi for the entire process, also I used nyanko json parameters as @G-Wang said in another [issue99](https://github.com/r9y9/deepvoice3_pytorch/issues/99)  it will work good.
I have formated cmu_arctic data  set just like ljspeech, Did pre-processing  and trained model using pre-trained ljspeech checkpoints. 
After synthesizing input text dosen't match to the output audio. 

Currenlty I'm trying with deepspeech3 json parameters, hope it'll work.
